src/pages/captured-pokemons/captured-pokemons-page.component.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PokemonsPage from 'Pages/pokemons-page/pokemons-page.component';
+
+import './captured-pokemons-page.styles.scss';
+
+const CapturedPokemonsPage = ({ collection }) => {
+  return (
+    <div>

```

Comment:
doesn't react 16 support `Fragment` component that let you to get rid of `<div>` wrapper?

Reply:
Well, if its a good pattern to always getting rid of these wrappers, so I'll be doing